src/assets/scripts/client/constants/themes.js

```diff
@@ -0,0 +1,5 @@
+import { THEME_DEFAULT } from './themes/default/theme';
```

Comment:
I like what you've done here.  This makes it easy to add new themes, keeps the overall code clean, and gives us on single `THEME` export with everything we need.

well done 👍 👍